src/TLuaInterpreter.cpp

```diff
@@ -314,6 +314,17 @@ int TLuaInterpreter::raiseEvent( lua_State * L )
             pE.mArgumentList.append( QString::number(lua_toboolean( L, i )) );
             pE.mArgumentTypeList.append( ARGUMENT_TYPE_BOOLEAN );
         }
+        else if( lua_isnil( L, i ) )
+        {
+            pE.mArgumentList.append( QString() );
+            pE.mArgumentTypeList.append( ARGUMENT_TYPE_NIL );
+        }
+        else
+        {
+            lua_pushstring( L, "raiseEvent: invalid argument type" );
```

Comment:
Could you apply the new Lua-style of error messages we've adopted here - see https://github.com/EldFitheach/Mudlet/blob/development/src/TLuaInterpreter.cpp#L5117 for an example.